bool   flag
uint32 id_robot
uint32 capacity